File: NET_8/FoodCall.Domain/Enums/OrderStatus.cs
public enum OrderStatus
{
    Pending,        // Aguardando confirmação do restaurante
    Confirmed,      // Restaurante aceitou
    Preparing,      // Na cozinha
    OutForDelivery, // Com o entregador
    Delivered,      // Entregue
    Cancelled       // Cancelado
}

File: Java/src/main/java/com/foodcall/domain/enums/OrderStatus.java
package com.foodcall.domain.enums;

public enum OrderStatus {
    PENDING,          // Aguardando confirmação do restaurante
    CONFIRMED,        // Restaurante aceitou
    PREPARING,        // Na cozinha
    OUT_FOR_DELIVERY, // Com o entregador
    DELIVERED,        // Entregue
    CANCELLED         // Cancelado
}
